File: Modul-5/Soal-4/PRAK504-2410817220012-NOOR-KHALISA.py
def reverse(angka):
    reverse_angka = 0
    while angka != 0:
        reverse_angka = reverse_angka * 10 + angka % 10
        angka = angka // 10
    return reverse_angka
def main():
    A, B = map(int, input("Input: ").split())
    A = reverse(A)
    B = reverse(B)
    C = A + B
    print(reverse(C))

if __name__ == "__main__":
    main()

File: Modul-5/Soal-4/PRAK504-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
int reverse(int angka){
    int reverse_angka = 0;
    while (angka != 0)
    {
        reverse_angka = reverse_angka * 10 + angka % 10;
        angka = angka / 10;
    }
    return reverse_angka;
}

int main(){
    int A, B;
    scanf("%d %d", &A, &B);
    A=reverse(A);
    B=reverse(B);
    int C = A + B;
    printf("%d", reverse(C));
    return 0;
}